src/main/rx/sbus.c

```diff
@@ -134,6 +140,30 @@ static void sbusDataReceive(uint16_t c, void *data)
             sbusFrameData->done = false;
         } else {
             sbusFrameData->done = true;
+
+#if defined(USE_TELEMETRY) && defined(USE_TELEMETRY_SBUS2)
+            switch(c) {
+                case 0x04: // sbus2, first telemetry page
+                case 0x14: // sbus2, second telemetry page
+                case 0x24: // sbus2, second telemetry page
+                case 0x34: // sbus2, second telemetry page
+                    if (c & 0x4)
```

Comment:
This condition seems to be true always. Is that correct?

Reply:
Good catch.